docs/upgrade_tails_usbs.rst

```diff
@@ -0,0 +1,124 @@
+Updating Tails USBs
+===================
+
+There are 3 components of SecureDrop that use Tails USBs:
+
+  #. The *Admin Workstation* (allowed to connect to the Internet)
+  #. The *Journalist Workstation* (allowed to connect to the Internet)
+  #. The *Secure Viewing Station* (airgapped, never allowed to connect to any network)
+
+You are responsible for keeping your Tails USBs updated. This guide will
+explain that process.
+
+One-Click Updates
+-----------------
+
+For the Internet-connected Tails workstations (the *Admin Workstation* and
+the *Journalist Workstation*), you'll see a notification when a new version
+of Tails is available. We recommend that you :ref:`back up your existing
+configuration<Step 1>` and upgrade as soon as possible.
+
+|Upgrade Notification|
+
+It usually takes some time for updates to download, so keep that in mind when choosing when to upgrade.
+
+.. |Upgrade Notification| image:: images/tails_upgrade_notification.png
+
+
+Manual Updates
+--------------
+
+However, because the *Secure Viewing Station (SVS)* is airgapped, it cannot
+receive upgrade notifications, so it will need to be updated manually.
+
+.. important:: 
+        The *SVS* stores some of SecureDrop's most sensitive data (decrypted submissions, the SecureDrop Application private key), so we **strongly** encourage you to manually upgrade the *SVS* whenever a new version of Tails is released.
+
+The Tails project releases updates `every 6 weeks`_. Occasionally they
```

Comment:
FAQ notwithstanding, in practice it's 6-8 weeks (it's tied to the [Firefox ESR schedule](https://wiki.mozilla.org/Release_Management/Calendar)). I would recommend linking to the Tails release calendar, here: https://tails.boum.org/contribute/calendar/

Reply:
Great- I've replaced the other link with this one (c91e4931f)